pymatgen/io/lammps/data.py

```diff
@@ -351,7 +351,7 @@ def get_string(self, distance=6, velocity=8, charge=4, hybrid=True):
             velocity (int): No. of significant figures to output for
                 velocities. Default to 8.
             charge (int): No. of significant figures to output for
-                charges. Default to 3.
+                charges. Default to 4.
```

Comment:
If there is no significant penalty in terms of size or performance, I suggest we default to 5 decimal places here. The only reason I say that is that for the TIP4P-FB water model I'm using, 5 decimals are required in order to keep the water molecule neutral. Obviously it's not a big deal to set this via kwarg if there are strong reasons to keep only 4 decimals as the default.

Reply:
The previous default was 3, changed to 4 as a safe buffer. Would you prefer to change it to even more? I am actually working on a charge writer class in `mdgo.forcefield`, it can auto-detect significant digits so that users don't have to specify that explicitly to make it right.